draft-ietf-rats-daa.md

```diff
@@ -79,54 +79,85 @@ A PKIX Certificate is an X.509v3 format certificate as specified by {{RFC5280}}.
-  |          |             .----------------------.      |
-  |          |         .-->+      Verifier        +--.   |
-  |          |         |   '----------------------'  |   |
-  |          |         |                             |   |
-  |          |         |Evidence          Attestation|   |
-  |          |         |                      Results|   |
-  |          |         |                             |   |
-  |      Cred|ential   |                             |   |
-  |          |         |                             |   |
-  |          v         |                             v   v
-  |        .-------------.                     .---------------.
-  '--------+  Attester   |                     | Relying Party |
-           '-------------'                     '---------------'
+    .--------.     .---------.       .--------.       .-------------.
+   | Endorser |   | Reference |     | Verifier |     | Relying Party |
+    '-+------'    | Value     |     | Owner    |     | Owner         |
+      |           | Provider  |      '---+----'       '----+--------'
+      |            '-----+---'           |                 |
+      |                  |               |                 |
+      | Endorsements     | Reference     | Appraisal       | Appraisal
+      |                  | Values        | Policy for      | Policy for
+      |                  |               | Evidence        | Attestation
+       '-----------.     |               |                 | Results
+                    |    |               |                 |
+               .----|----|---------------|-----------------|------.
+               |    |    |               |                 |      |
+               |    v    v               v                 |      |
+               |  .-------------------------.              |      |
+         .------->|         Verifier        +-----.        |      |
+        |      |  '-------------------------'      |       |      |
+        |      |                                   |       |      |
+        |  Evidence                    Attestation |       |      |
+        |      |                       Results     |       |      |
+        |      |                                   |       |      |
+        |      |                                   v       v      |
+  .-----+----. |                               .---------------.  |
+  | Attester | |                               | Relying Party |  |
+  '----------' |    DAA Issuer                 '---------------'  |
+               '--------------------------------------------------'
+~~~~
+{: #join-mapping title="RATS Architecture for the Join Protocol"}
+
+The Join Protocol is essentially an enrollment protocol that consumes Evidence from the Attester (therefore the mapping to the Verifier role). Appraisal Policies for Evidence specifically dedicated to that appraisal procedure are used to decide whether to issue a DAA credential to an Attester or not.
```

Comment:
My opinion is that the Attestation Results (not the Evidence per se) is what is used to decide whether to issue a DAA credential to an Attester.   Conceptually, the Evidence is verified, and the result (which may simply be an API call) passes the Attestation Result to the component issuing the credential.

Reply:
Yes. That was me not paying enough attention to the issue. How about this new text?

```suggestion
The Join Protocol is essentially an enrollment protocol that consumes Evidence from the Attester (therefore the mapping to the Verifier role). Corresponding Appraisal Policies for Evidence specific to the Join Protocol are used to produce Attestation Results to decide whether to issue a DAA credential to an Attester or not (therefore the mapping to the Relying Party role).
```

Reply:
Looks good, thanks!